apps/web/src/components/templates/email-editor/EmailInboxContent.tsx

```diff
@@ -34,7 +35,7 @@ export const EmailInboxContent = ({
               margin: '5px 0px',
             }}
           >
-            {integration ? integration?.credentials?.from : 'No active email integration'}
+            <EmailIntegrationInfo integration={integration} field={'from'} />
```

Comment:
As the `field` default value is already `from` we wouldn't need to set it up explicitly. But I personally find it helpful, so this is ok for me.

Reply:
I know but I also find it helpful :)